README.md

```diff
@@ -1,196 +1,11 @@
 ## Overview
 
-**USED WITH SPOTLIGHT 1.4**
```

Comment:
This documentation moved to the Wiki at [Data Integrity Scripts](https://github.com/elrayle/spotlight_utils/wiki/Data-Integrity-Scripts).